apps/notifications/src/panel/state/selectors/get-all-notes.js

```diff
@@ -11,8 +6,29 @@ import getNotes from './get-notes';
 let prevAllNotes;
 let sortedNotes = [];
 
-const noteId = ( { id } ) => id;
-const parsedTimestamp = ( { timestamp } ) => Date.parse( timestamp );
+const byTimestamp = ( a, b ) => {
+	const difference = Date.parse( a.timestamp ) - Date.parse( b.timestamp );
+	if ( difference < 0 ) {
+		return -1;
+	}
+
+	if ( difference > 0 ) {
+		return 1;
+	}
+
+	return 0;
+};
```

Comment:
Seems like this should work the same way, no?

```suggestion
const byTimestamp = ( a, b ) => Date.parse( a.timestamp ) - Date.parse( b.timestamp );
```

Also, if these are timestamps, do we really need to parse them first before comparing them as numbers?